osd/user-ca-bundle_invalid.json

```diff
@@ -0,0 +1,8 @@
+{
+    "severity": "Info",
+    "service_name": "SREManualAction",
+    "cluster_uuid": "${CLUSTER_UUID}",
+    "summary": "Action required: Misconfigured additional Trusted CA bundle",
+    "description": "Your cluster requires you to take action because the Additional Trusted CA Bundle that you have applied to your cluster is corrupt or cannot be correctly parsed. This may impact your cluster's ability to communicate with any external resources that require that bundle, including those accessible via a cluster-wide proxy. Please remove or update the Additional Trusted CA Bundle to ensure it is in a parseable, plaintext PEM-formatted format. If you require any further assistance, please file a support request.",
```

Comment:
I think @aweiteka did remove the mention to file a support ticket because it's in the base template (see here https://github.com/openshift/managed-notifications/pull/125) 
so you can probably remove it too unless I missed something

Reply:
Thanks! I've removed that part now.